ansible/roles_ocp_workloads/ocp4_workload_fuse_online/tasks/workload.yml

```diff
@@ -152,7 +156,8 @@
   k8s:
     state: present
     namespace: "{{ ocp4_workload_fuse_online_project }}"
-    definition: "{{ lookup('template', role_path ~ '/templates/coolstore-catalog-mongodb-persistent.j2' ) | from_yaml }}"  
+    definition: >-
+      {{ lookup('template', role_path ~ '/templates/coolstore-catalog-mongodb-persistent.j2' ) | from_yaml }}
```

Comment:
We can shorten this more. Ansible lookups default to looking in the role path templates directory, so we can just use:
```
    definition: "{{ lookup('template', 'coolstore-catalog-mongodb-persistent.j2' ) | from_yaml }}"  

```